src/main/java/frc/robot/subsystems/BallPath/Shooter/PIDShooterImpl.java

```diff
@@ -318,12 +294,15 @@ public boolean blocking() {
         double absoluteWheelSpeed = Math.abs(shooterEncoderReadingVelocity);
         boolean spinningDown = setPointShooterPID == 0 && absoluteWheelSpeed > threshold;
         return spinningUp || spinningDown;
-        // return false;
     }
 
     @Override
     public boolean readyToShoot(){
-        return turretReady && hoodReady && shooterPid.atSetpoint();
+        boolean shooterReady = false;
+        if(Math.abs(shooterEncoderReadingVelocity) > setPointShooterPID - 500 && Math.abs(shooterEncoderReadingVelocity) < shooterEncoderReadingVelocity + 500){
+            shooterReady = true;
+        }
+        return turretReady && shooterReady;
```

Comment:
Why don't we wait for `hoodReady` here anymore?

Reply:
also a bartek question